docs/source/gateway.md

```diff
@@ -0,0 +1,118 @@
+# Gateway
+
+## Overview
+
+The Gateway is a new service introduced in v2.4 that provides a simplified, minimal API for submitting transactions
+to a Fabric network.
+
+## Writing client applications
+
+Starting with v2.4, client applications should use one of the v2.4 SDKs which are optimised to interact with the Gateway.
+These SDKs expose the same high-level programming model which was first introduced with v1.4 of Fabric.
+
+The gateway supports the following actions
+- **Evaluate** (query) a transaction.  This will invoke a smart contract (chaincode) function on a single peer and return the result
```

Comment:
True.  I've changed it to:
   - **Evaluate** a transaction proposal.  This will invoke a smart contract (chaincode) function on a single peer and return the result to the client.  This is typically used to query the current state of the ledger without making any ledger updates.